ui/src/app/views/homepage/component.tsx

```diff
@@ -252,16 +255,23 @@ export const Homepage = withUserProfile()(class extends React.Component<
     }
   }
 
+  async checkUseBillingProjectBufferFeatureFlag() {
+    const config = await configApi().getConfig();
```

Comment:
This is pretty rough, there's an existing instance of this on the homepage too.

1. The server config is already loaded on the client side and it blocks the application load.
2. This config can be considered to be static for the lifetime of the app's client session.

I think we need a `configStore`, or something even more streamlined for synchronous access; this PR is kind of forcing the issue for me. @blrubenstein I don't know whether this has come up on other React PRs already (I'm surprised if not).

Reply:
Do we have a pattern for centrally storing application data? I'm thinking something like redux

Reply:
Here is our current pattern - it's not super elegant and it's led to a few bugs and some issues during testing. In this case extending the current pattern is probably sufficient. We should probably revisit once we're fully on React:

https://github.com/all-of-us/workbench/blob/00a906d0e936cead8ca2ea7cf5b63cdf324e86cc/ui/src/app/utils/navigation.tsx#L14

Note: we do use redux in the cohort builder in a few places.